packages/framework/src/Model/Advert/AdvertPositionRegistry.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Shopsys\FrameworkBundle\Model\Advert;
+
+class AdvertPositionRegistry
+{
+    const POSITION_HEADER = 'header';
+    const POSITION_FOOTER = 'footer';
+    const POSITION_PRODUCT_LIST = 'productList';
+    const POSITION_LEFT_SIDEBAR = 'leftSidebar';
+
+    public function getTranslatedPositionDescriptionsIndexedByPositionName(): array
```

Comment:
You've omitted all annotations. Why?

Reply:
Also, the method's name is a bit mouthful. Wouldn't `getAllLabelsIndexedByNames` suffice?

Reply:
I though that the typehint is sufficient but I forgot that it is not specific enough, so I added the `string[]` annotation :slightly_smiling_face: 

You are right about the name :laughing: I changed it :+1: